commentHandler.go

```diff
@@ -308,3 +362,19 @@ func checkTransition(requestedAction string, currentState string) (string, bool)
 
 	return "", false
 }
+
+func removeMilestone(client *github.Client, ctx context.Context, URL string) error {
```

Comment:
Please add a comment and a link to the GitHub issue which gave this solution/work-around.

In the future someone may delete the code and revert to the library usage which we know currently has a bug.